components/glyphicons/glyphicons-en.html

```diff
@@ -0,0 +1,810 @@
+---
+title: Glyphicons
+description: Documentation and working examples for Glyphicons
+language: en
+altLangPage: glyphicons-fr.html
+dateModified: 2023-07-06
+layout: no-container
+---
+<div class="container">
+	<h1>{{ page.title }}</h1>
+
+	<span class="wb-prettify all-pre hide"></span>
+
+	<!--<h2>Purpose</h2>
+	<p>This adds Bootstrap Glyphicons to a Canada.ca page. Bootstrap includes 260 glyphs from the Glyphicon Halflings set. Glyphicons Halflings are normally not available for free, but their creator has made them available for Bootstrap free of cost. As a thank you, you should include a link back to <a href="https://glyphicons.com/">Glyphicons</a> whenever possible.</p>-->
+
+	<h2>Working examples</h2>
```

Comment:
This heading don't have any sibling, please remove unless if you are adding a sibling for the glyphicon error state addition.
```suggestion
```